Remove serde_bytes from dev-dependencies to resolve circular dependency issue.
--- a/Cargo.toml	1969-12-31 16:00:00.000000000 -0800
+++ b/Cargo.toml	2019-06-28 10:38:20.312246679 -0700
@@ -46,9 +46,6 @@
 version = "0.3"
 features = ["stable"]
 
-[dev-dependencies.serde_bytes]
-version = "0.10"
-
 [dev-dependencies.serde_derive]
 version = "1.0"
 
